utils.py

```diff
@@ -51,7 +53,10 @@ def iso_format_validation(timestamp):
             iso = '%Y-%m-%dT%H:%M:%SZ'
             timestamp = datetime.utcfromtimestamp(timestamp).strftime(iso)
         except ValueError:
-            log.error("Error. Timestamp is not is ISO-8601 format.")
-            return 400
-
+            log.error('Error. Timestamp is not is ISO-8601 format.')
+            return False
+        except OverflowError:
+            log.error('Error. Timestamp float value is too large to be used ',
+                      'as datetime.')
```

Comment:
Add `timestamp` value to log. May use log.debug instead of error, as it's expected for a validation function to receive invalid values.